src/main/java/com/fontys_automotive/api/tus/TusController.java

```diff
@@ -164,17 +148,98 @@ public ResponseEntity<?> postTeachers(Teacher teacher){
         return new ResponseEntity<>(result, HttpStatus.OK);
     }
 
+    @GetMapping("/job")
+    public ResponseEntity<?> postJob(){
+
+        URL url = null;
+        {
+            try {
+                url = new URL("https://import.8vance.com/data_import/structured_data_import_for_job/");
```

Comment:
Probeer de URL op te halen via een property ipv overal plain in te vullen
```java
@Value("${8vance.endpoint}")
private String uri;
```
waarbij `8vance.endpoint` de URL is in de `applications.properties`